roles/kubernetes/tasks/cleanup.yml

```diff
@@ -0,0 +1,11 @@
+---
+# This play contains tasks for cleaning up kubernetes
+
+- name: stop kubernetes
+  service: name=kubernetes state=stopped
+
+- name: cleanup iptables for kubernetes
```

Comment:
yes, I agree. Let me see if I can make use of a common template of some sort.

Let me track this separately so I can fix all roles together.